WindowsFormsApp3/Form1.cs

```diff
@@ -167,15 +171,13 @@ void CreateFromWorld(IWorldspaceGetter wrld)
             bitmap.Save(@"C:\debug\tamriel.bmp");
         }
 
-        public float[,] ParseHeights(Noggog.ReadOnlyMemorySlice<byte> input_in)
+        public float[,] ParseHeights(ReadOnlyMemorySlice<byte> input)
```

Comment:
Spans are basically just arrays that can only live on the stack.. and are able to slice and pass a subsection without allocating/copying to a new array.

Upgraded the usage a bit to avoid an array allocation by just using the span more directly